proto/p4/v1/p4runtime.proto

```diff
@@ -405,15 +405,19 @@ message PacketReplicationEngineEntry {
 
 // Used for replicas created for cloning and multicasting actions.
 message Replica {
-  uint32 egress_port = 1;
+  oneof port_kind {
```

Comment:
Is the `oneof` necessary here given that `egress_port` is deprecated? It shouldn't be harmful, just wondering what the best practice here is.

Reply:
That's a good point, but I'm just sticking to what we did for watch port.